src/tsickle.ts

```diff
@@ -779,6 +779,12 @@ class ExternsWriter extends ClosureRewriter {
   /** visit is the main entry point.  It generates externs from a ts.Node. */
   public visit(node: ts.Node, namespace: string[] = []) {
     switch (node.kind) {
+      case ts.SyntaxKind.SourceFile:
```

Comment:
is this intentionally part of this PR? Not a big deal, just making sure you are aware.